migrations/2018-12-08-194435_create_users/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE users (
+  id INT PRIMARY KEY,
+  username VARCHAR NOT NULL,
+  password VARCHAR NOT NULL,
+  hash VARCHAR NOT NULL
```

Comment:
What is `hash` used for?

Reply:
This is a typo I believe. It was intended to be the hash of the password...but we are also storing that. I think I only need password. I will update and remove the hash field. Migrations will get updated too.